ui/src/pages/Dashboard/DataIngestionModal.tsx

```diff
@@ -0,0 +1,72 @@
+import Modal from '@splunk/react-ui/Modal';
+import React, { ReactElement } from 'react';
+import styled from 'styled-components';
+
+import { variables } from '@splunk/themes';
+import Button from '@splunk/react-ui/Button';
+import { StyledButton } from '../EntryPageStyle';
+
+const ModalWrapper = styled(Modal)`
+    width: 60%;
+    height: 80%;
```

Comment:
does it work fine on small and huge screen sizes?

Reply:
Changed it to vh/vw respectivley